src/core/Akka/Actor/ActorCell.Children.cs

```diff
@@ -385,15 +385,14 @@ public IInternalActorRef GetSingleChild(string name)
         /// </summary>
         /// <param name="child">TBD</param>
         /// <returns>TBD</returns>
-        protected SuspendReason RemoveChildAndGetStateChange(IActorRef child)
+        protected SuspendReason? RemoveChildAndGetStateChange(IActorRef child)
```

Comment:
`SuspendReason` will be `null` if this child was not found - should never happen but it is theoretically possible.